tests/e2e/utils/kubeflow_installation.py

```diff
@@ -0,0 +1,200 @@
+                build_retcode = build_certmanager()
+                assert build_retcode == 0
+            elif component_name == "aws-load-balancer-controller":
+                build_retcode = build_alb_controller(cluster_name)
+                assert build_retcode == 0
+            else:
+                install_helm(component_name, installation_path, namespace)
+        # kustomize
+        else:
+            if "crd_required" in path_dic[component_name]:
+                crd_required = path_dic[component_name]["crd_required"]
+                crd_meet = False
+            for kustomize_path in installation_path:
+                if not crd_meet:
+                    apply_kustomize(kustomize_path, crd_required)
+                    crd_meet = True
+                apply_kustomize(kustomize_path)
+
+    if "validations" in path_dic[component_name]:
+        identifier = path_dic[component_name]["validations"]["identifier"]
+        common_label = path_dic[component_name]["validations"]["common_label"]
+        namespace = path_dic[component_name]["validations"]["namespace"]
+        print(f"Waiting for {component_name} pods to be ready ...")
+        retcode = kubectl_wait_pods(common_label, namespace, identifier)
+        assert retcode == 0
+    print(f"All {component_name} pods are running!")
+
+
+def build_certmanager():
+    retcode = subprocess.call(
+        f"helm repo add jetstack https://charts.jetstack.io".split()
+    )
+    assert retcode == 0
+    retcode = subprocess.call(f"helm repo update".split())
+    assert retcode == 0
+    cmd = f"helm install cert-manager jetstack/cert-manager \
+                        --namespace cert-manager \
+                        --create-namespace \
+                        --version v1.9.1 \
+                        --set installCRDs=true".split()
+    return subprocess.call(cmd)
+
+
+def build_alb_controller(cluster_name):
+    retcode = subprocess.call(
+        f"helm repo add eks https://aws.github.io/eks-charts".split()
+    )
+    assert retcode == 0
+    retcode = subprocess.call(f"helm repo update".split())
+    assert retcode == 0
+    cmd = f"helm install aws-load-balancer-controller eks/aws-load-balancer-controller \
+            -n kube-system \
+            --set clusterName={cluster_name} \
+            --set serviceAccount.create=false \
+            --set serviceAccount.name=aws-load-balancer-controller \
+            --version v2.4.3".split()
+    return subprocess.call(cmd)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    INSTALLATION_OPTION_DEFAULT = "kustomize"
+    parser.add_argument(
+        "--installation_option",
+        type=str,
+        default=INSTALLATION_OPTION_DEFAULT,
+        help=f"Kubeflow Installation option (helm/kustomize), default is set to {INSTALLATION_OPTION_DEFAULT}",
+        required=False,
+    )
+    AWS_TELEMETRY_DEFAULT = "enable"
+    parser.add_argument(
+        "--aws_telemetry_option",
```

Comment:
nit: rename to enable_telemetry with boolean true/false instead of string

Reply:
will do

Reply:
addressed.